pyramid_oereb/lib/renderer/extract/templates/xml/article.xml

```diff
@@ -4,7 +4,7 @@
     <data:Number>${article.number}</data:Number>
     %if article.text:
     <data:Text>
-        <%include file="multilingual_m_text.xml" args="text=article.text"/>
+        <%include file="multilingual_text.xml" args="text=article.text"/>
```

Comment:
Should this be in this PR?

Reply:
Thanks for your review, you are right this is not directly linked to the PR - I noticed there were three templates doing the same thing so I did some refactoring (50b988db5bda464a2b6d033a4f63b72c7409931f)